lib/puma/binder.rb

```diff
@@ -14,6 +13,8 @@ class Binder
 
     RACK_VERSION = [1,6].freeze
 
+    @@minissl_loaded = false
```

Comment:
Just my opinion, but could you avoid class variables (and use an `@ivar` on the class instead)?
I consider them kind of deprecated and they're pretty slow on all Ruby implementations.

Reply:
Alternatively, would checking with `defined?(MiniSSL)` or some other constant work and not need explicit state?

Reply:
I thought most of the class variable issues related to their use in 'subclasses', but the constant would work fine.

Reply:
Agreed with @eregon

Reply:
Updated to use `load_ssl unless defined? ::Puma::MiniSSL::ContextBuilder`.  Using a constant bothers RuboCop

There are CI failures that seem unrelated to this PR, trying to determine the cause...